### DISCUSSION TITLE
how to make these (unavailable) items works in Tray Icons menu?

### DISCUSSION THREAD

**Discussion by rsyqvthv** [winsiderss/systeminformer#1111] [Questions / Answers] [answered]
<img width="391" src="https://user-images.githubusercontent.com/2230369/151505392-4dc3b112-c28b-49af-aaf6-287651c723d9.png">

as shown in the screenshot above, there are couple of items status unavailable, in which I would like to show the "Network usage (text)", don't know how to make it works...

**Comment by Biswa96** [comment, marked as answer]
Run Process Hacker as administrator. Or click on the Hacker > Show details for all processes option which will re-run Process Hacker as administrator.

**Reply by rsyqvthv** [reply]
thank you very much! :)
